go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/operator-framework/operator-lifecycle-manager v0.17.0
 	github.com/pkg/errors v0.9.1
 	k8s.io/api v0.18.9
+	k8s.io/apiextensions-apiserver v0.18.9
```

Comment:
Let's remove this line, it is only used in the test instead of product code.

Reply:
Sure, after deleting `testbin`, `go.mod` does not include this module.